src/main/java/core/basesyntax/service/reader/ReaderService.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service.reader;
+
+import java.util.List;
+
+public interface ReaderService {
```

Comment:
Don't segregate your interfaces too much. Combine wirte to file and read from file in one FileService

Reply:
agree, fixed